index.html

```diff
@@ -233,7 +359,52 @@ <h2 class="conta">CONTACT US</h2>
 <li class="M-icon">
 <i class="fa-solid fa-m"><a href="https://twitter.com/steady1700"></a></i></li>
 </ul>
+
 <div class="my-mail">emailhere@email.com</div>
+
+    </div>
+
+    <div class="contact-D">
+        <h2 class="conta">Let's chat!</h2>
+    <p class="cont-text">
+        if you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
+    </p>
+        <form action="https://formspree.io/f/mlezygyr" method="post">        
+             
+               <input type="text" id="name-D" name="user_name" placeholder="  Name" maxlength="30" required>
+    
+               <input type="email" id="mail-D" name="user_email" placeholder="  Email Address" required>
+             
+             
+               <textarea cols="33" rows="16"  id="msg-D" name="user_message" 
+               placeholder=" 
+     Write your message here" maxlength="500" required ></textarea>
+            
+             <div class="button">
+               <button type="submit" id="bttn-D"> send </button>
+             </div>
```

Comment:
- [ ] **Required:** Great work on the form it's almost done 👏🏽 . Just minor color changes - When a user focus on the input fields, the color should change and matches the design. here are some pictures for your reference:

## Expected Look
![image](https://user-images.githubusercontent.com/35315311/178088763-e2d6acfa-c1cd-4ef4-953e-ef1ccae8804c.png)

## Your current design (Looks like you're using black 👀 )

![image](https://user-images.githubusercontent.com/35315311/178088808-0444d53a-3b88-4fe5-a903-b4d63ed7339f.png)